### ISSUE TITLE
grab_access_token vs grab_refresh_token

### ISSUE THREAD

**Issue by Aspire1Inspire2** [areed1192/td-ameritrade-python-api#133]
In the doc string, both of them are described as `"Refreshes the current access token."`
Must be something wrong here.

**Comment by Aspire1Inspire2** [comment]
Name | Values | Description
-- | -- | --
grant_type                                                                                                                                                                             (required) |   | The grant type  of the oAuth scheme. Possible values are authorization_code,  refresh_token
refresh_token |   | Required if using refresh token grant
access_type |   | Set to offline  to receive a refresh token on an authorization_code grant type request.  Do not set to offline on a refresh_token grant type request.

In the `grab_refresh_token` function, it is using:
```
data = {
            'client_id': self.client_id,
            'grant_type': 'refresh_token',
            'access_type': 'offline',
            'refresh_token': self.state['refresh_token']
        }
```
However TDAmeritrade clearly says that `refresh_tocken` `grant_type` is not to be used together with `offline` `access_type`.

Could you check the function doc string and implementation correctness?
I am wondering what is the expected behavior of `grab_refresh_token` function?
Is it functioning as expected?

**Comment by areed1192** [comment]
What I decided to do in the latest update is to have one method dedicated specifically to exchanging an authorization code for an access token and refresh token. That method is called `exchange_code_for_token`. I then made a method dedicated to grabbing a new refresh token called `grab_refresh_token` which would include the `offline` argument. I also included the `grab_access_token` which removes the `offline` argument.

From what I've seen, if you request a new refresh token before the old one is expired, then you shouldn't have to go through the login process again. I'm going to update the docstrings to make it a little more apparent and clearer how each of the functions are different.
